playtron-os-files/playtron-os-files.spec

```diff
@@ -30,9 +30,10 @@ cp playtron-os-files-%{version}/LICENSE %{buildroot}/usr/share/licenses/playtron
 /usr/bin/clatd-ipv6-check
 /usr/bin/create-swap.sh
 /usr/bin/hwctl
```

Comment:
if the goal is to have the files in alphabetical order, shouldn't all `/usr/bin` files be listed before files in `/usr/lib`?

Reply:
I need new glasses lol. That was my bad, let me update that.